### PR TITLE
[Management] Make ``BrokerBase#getOwnedNamespaces`` to pure async.

### PR DESCRIPTION
**Pull request by mattisonchao**
### Motivation

Make ``BrokerBase#getOwnedNamespaces`` to pure async.

### Modifications

- Make ``BrokerBase#getOwnedNamespaces`` to pure async.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### Documentation

- [x] `no-need-doc`

### CODE REVIEW

**Comment by mattisonchao** [comment]
@codelipenghui  @eolivelli  @Technoboy- @Jason918 @Shoothzj @RobertIndie @nodece 

PTAL, when you have time. :)

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java [hunk 1]
```diff
@@ -116,21 +111,15 @@ public void getActiveBrokers(@Suspended final AsyncResponse asyncResponse,
...
-                .exceptionally(ex -> {
-                    Throwable realCause = FutureUtil.unwrapCompletionException(ex);
-                    LOG.error("[{}] Failed to get the information of the leader broker.", clientAppId(), realCause);
-                    if (realCause instanceof WebApplicationException) {
-                        asyncResponse.resume(realCause);
-                    } else {
-                        asyncResponse.resume(new RestException(realCause));
-                    }
-                    return null;
+                    log.info("[{}] Successfully to get the information of the leader broker.", clientAppId());
```
Comment (Technoboy-) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java:118]: Add additional log, right ?

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java [hunk 2]
```diff
@@ -142,20 +131,21 @@ public void getLeaderBroker(@Suspended final AsyncResponse asyncResponse) {
...
-            LOG.error("[{}] Failed to get the namespace ownership status. cluster={}, broker={}", clientAppId(),
-                    cluster, broker);
-            throw new RestException(e);
-        }
+        validateSuperUserAccessAsync()
+                .thenCompose(__ -> validateClusterOwnershipAsync(cluster))
+                .thenAccept(__ -> validateBrokerName(broker))
+                .thenCompose(__ -> pulsar().getNamespaceService().getOwnedNameSpacesStatusAsync())
+                .thenAccept(ownedNamespaces -> {
+                    log.info("[{}] Successfully to get the owned namespaces.", clientAppId());
```
Comment (Technoboy-) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java:143]: And add log here.

**Comment by mattisonchao** [comment]
@Technoboy-  PTAL :)

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java [hunk 3]
```diff
@@ -140,20 +140,22 @@ public void getLeaderBroker(@Suspended final AsyncResponse asyncResponse) {
...
-                    cluster, broker);
-            throw new RestException(e);
-        }
+    public void getOwnedNamespaces(
+            @Suspended AsyncResponse asyncResponse,
+            @PathParam("clusterName") String cluster,
+            @PathParam("broker-webserviceurl") String broker){
+        validateSuperUserAccessAsync()
+                .thenCompose(__ -> validateClusterOwnershipAsync(cluster))
+                .thenAccept(__ -> validateBrokerName(broker))
```
Comment (Jason918) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java:149]: thenAccept --> thenCompose?

Comment (mattisonchao) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java:149]: it's not an async method, I think ``thenAccept`` is ok.

**Review by Technoboy- — APPROVED** [review]

**Review by jiazhai — APPROVED** [review]

**Review by Jason918 — APPROVED** [review]

**Pull request closed without merging**